scripts/process/clean_county_boundaries.R

```diff
@@ -0,0 +1,255 @@
+#' Cleans data for historical county polygons.
+process.county_boundaries <- function(viz){
+  deps <- readDepends(viz)
+  
+  # unzip the shapefiles, which are zip files within a zip file
+  map_zip <- deps$county_boundaries_zip
+  map_dir <- file.path(tempdir(), 'county_boundaries')
+  unzip(map_zip, exdir=map_dir)
+  map_shp_zips <- dir(dir(map_dir, full.names=TRUE), full.names=TRUE)
+  map_shps <- lapply(map_shp_zips, function(zipname) {
+    files <- unzip(zipname, exdir=map_dir) # list of vectors of dbf/prj/shp/shp.xml/etc filepaths, length = number of shapefiles
+    base_no_ext <- gsub('\\.zip', '', basename(zipname))
+    year <- substring(base_no_ext, nchar(base_no_ext)-3)
+    list(files=files, year=year)
+  })
+  names(map_shps) <- sapply(map_shps, function(map_shp) map_shp$year)
+  
+  # load into R
+  decade_shps <- lapply(map_shps, function(shapefile_files) {
+    message("reading county data for year ", shapefile_files$year)
+    shp_file <- grep("\\.shp$", shapefile_files$files, value=TRUE)
+    layer <- sf::st_layers(shp_file)$name
+    one_year_sf <- sf::st_read(shp_file, layer=layer, stringsAsFactors=FALSE, quiet=TRUE, check_ring_dir=TRUE)
```

Comment:
I'm running this right now and I'm getting an error on this line for the very first file (1950):
```
one_year_sf <- sf::st_read(shp_file, layer=layer, stringsAsFactors=FALSE, quiet=TRUE, check_ring_dir=TRUE)
Error in st_sf(x, ..., agr = agr) : 
  no simple features geometry column present
```
Updated my `sf` package from 0.5 something to 0.6-0 and that went away (also updated `sp` to 1.2-7, but don't remember what the prev number was)